community/kernel/src/main/java/org/neo4j/kernel/impl/cache/MeasureDoNothing.java

```diff
@@ -21,6 +21,10 @@
 
 import org.neo4j.logging.Log;
 
+/**
+ * @deprecated Remove this nonsense and use standard beans for getting GC measurements
+ */
+@Deprecated
```

Comment:
It's measuring all kinds of stalls, not just stalls caused by GC. An interesting case is for instance documented at: http://www.evanjones.ca/jvm-mmap-pause.html
I don't think this is public API, though, so you can remove it without deprecating.

Reply:
This thing will be replaced by 2 things: one that reports real gc pauses and another one that reports stalls.

Reply:
This one is replaced with #11010